wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/pdf/net/PdfExportRepository.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021 Ubique Innovation AG <https://www.ubique.ch>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+package ch.admin.bag.covidcertificate.wallet.pdf.net
+
+import android.content.Context
+import ch.admin.bag.covidcertificate.common.BuildConfig
+import ch.admin.bag.covidcertificate.sdk.android.CovidCertificateSdk
+import ch.admin.bag.covidcertificate.sdk.android.data.Config
+import ch.admin.bag.covidcertificate.sdk.android.net.CertificatePinning
+import ch.admin.bag.covidcertificate.sdk.android.net.interceptor.JwsInterceptor
+import ch.admin.bag.covidcertificate.sdk.android.net.interceptor.UserAgentInterceptor
+import ch.admin.bag.covidcertificate.sdk.android.utils.SingletonHolder
+import ch.admin.bag.covidcertificate.sdk.core.models.healthcert.CertificateHolder
+import okhttp3.Cache
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+class PdfExportRepository private constructor(spec: PdfExportSpec) {
```

Comment:
If I don't miss something, I think that in this case, the constructor argument probably might be reduced to context only. The base url (TRANSFORMATION) in the BuildConfig should also be accessible on line 52.